main.c

```diff
@@ -547,14 +573,14 @@ int init() {
   }
 
   // Create window
-  window = SDL_CreateWindow("Zombie Breakout", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, WINDOW_W, WINDOW_H, SDL_WINDOW_RESIZABLE);
+  window = SDL_CreateWindow("Zombie Breakout", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, 640, 480, SDL_WINDOW_RESIZABLE);
   if (window == NULL) {
     printf("Window could not be created! SDL Error: %s\n", SDL_GetError());
     return 0;
   }
 
   // Create renderer for window
-  renderer = SDL_CreateRenderer(window, -1, SDL_RENDERER_ACCELERATED);
+  renderer = SDL_CreateRenderer(window, -1, SDL_RENDERER_SOFTWARE);
```

Comment:
This should only be done on Xbox (`#if defined(NXDK)`), so other platforms are not slowed down.

*(Same as window flags)*